web/themes/contrib/civictheme/civictheme.api.php

```diff
@@ -59,4 +60,37 @@ function hook_civictheme_automated_list_preprocess_view_alter(array &$variables,
     $variables['with_background'] = TRUE;
     $variables['vertical_spacing'] = 'both';
   }
+
+  /**
+   * Allow to suppress page regions for pages with Layout Builder enabled.
+   *
+   * @param array $variables
+   *   Array of variables passed to the page template.
+   * @param array $context
+   *   Array of context data.
+   *   - node: The node object.
+   *   - layout_builder_settings_per_view_mode: An array of the layout builder
+   *     settings keyed by view mode.
+   *
+   * @SuppressWarnings(PHPMD.StaticAccess)
+   */
+  function hook_civictheme_layout_suppress_page_regions_alter(array &$variables, array $context): void {
```

Comment:
new hook to control "conflicting" theme and layout regions

![Layouts](https://github.com/civictheme/monorepo-drupal/assets/378794/27927501-c706-4726-8581-e7d3c51be3e8)